packages/expo-task-manager/ios/EXTaskManager/EXTaskExecutionRequest.m

```diff
@@ -0,0 +1,64 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+#import <EXTaskManager/EXTaskExecutionRequest.h>
+
+@interface EXTaskExecutionRequest ()
+
+@property (nonatomic, strong, nullable) NSMutableSet<id<EXTaskInterface>> *tasks;
+@property (nonatomic, strong, nullable) NSMutableArray<id> *results;
+@property (nonatomic, assign) int tasksCount;
+
+@end
+
+
+@implementation EXTaskExecutionRequest
+
+- (instancetype)initWithCallback:(void(^)(NSArray *results))callback
+{
+  if (self = [super init]) {
+    _callback = callback;
+    _tasks = [NSMutableSet new];
+    _results = [NSMutableArray new];
+  }
+  return self;
+}
+
+- (void)addTask:(nonnull id<EXTaskInterface>)task
+{
+  if (task != nil) {
```

Comment:
```suggestion
  if (task) {
```

Reply:
and you could also remove `nonnull` annotation from `addTask:` if you handle this case? idk